roles/confluent.test/upgrade_testing/mtls-customcerts-rhel-540-current.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+## variables
+
+SCENARIO_NAME=mtls-customcerts-rhel
+
```

Comment:
Can this get merged into 5.5.0-post and pint merged up so that we can test an upgrade to any version?
You could save a variable for current branch:

```suggestion
CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)

...

echo "Checkout $CURRENT_BRANCH branch"
(cd ../ && git checkout $CURRENT_BRANCH)
```